tests/unit/recommenders/evaluation/test_python_time.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@simonzhaoms @ChuyangKe if we are going to start these kinds of tests as a new test pattern, what name would you suggest we use? "time performance tests"? other?

Reply:
I think time performance tests is good

Reply:
Then maybe @chuyangke, should we rename the file to `test_python_evaluation_time_performance.py` or `test_time_performance_python_evaluation.py` or similar? 

Whatever we choose now, we should replicate the pattern in new files

Reply:
Then maybe @chuyangke, should we rename the file to `test_python_evaluation_time_performance.py` or `test_time_performance_python_evaluation.py` or similar? 

Whatever we choose now, we should replicate the pattern in new files

Reply:
Let's use `test_python_evaluation_time_performance.py` then.